SDK/MobileCenterPush/Microsoft.Azure.Mobile.Push.UWP/Push.cs

```diff
@@ -81,14 +81,14 @@ private void ApplyEnabledState(bool enabled)
 
                                 // Send channel URI to backend
                                 MobileCenterLog.Debug(LogTag, $"Push token '{pushToken}'");
-                                var pushInstallationLog = new PushInstallationLog(null, null, pushToken, Guid.NewGuid());
-                                await Channel.EnqueueAsync(pushInstallationLog).ConfigureAwait(false);
                             }
                             else
                             {
                                 MobileCenterLog.Error(LogTag, "Push service registering with Mobile Center backend has failed.");
                             }
                         }
+                        var pushInstallationLog = new PushInstallationLog(null, null, pushToken, Guid.NewGuid());
```

Comment:
This could be executed after SDK disabled now, better fix would just be remove await as we don't need to wait here, await was just to disable the warning.